Dear ImGui Sample/ImGuiController.cs

```diff
@@ -588,4 +564,132 @@ public static void CheckGLError(string title)
             }
         }
     }
-}
+
+    public static class KeyTranslator
+    {
+        public static ImGuiKey Translate(Keys key)
+        {
+            switch (key)
```

Comment:
[optional] you can reduce the length a bit if you do something like this.
https://github.com/zaafar/ClickableTransparentOverlay/blob/master/ClickableTransparentOverlay/ImGuiInputHandler.cs#L161:L172